### PR TITLE
Bias-free communication

### LINKED ISSUE #33705: Bias-free communication sweep
It's a good idear to scan the docs every few years for bia-free communication problems.

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #33705

IMO, it's a good i-dear to do this every few years to fix problems. This was on my UE tracking issue for Blazor docs, but it's something that I usually do for the whole repo, given that it's just quick search filter stuff.

* https://learn.microsoft.com/en-us/style-guide/bias-free-communication
* https://learn.microsoft.com/en-us/style-guide/militaristic-language

Notes ...

* I left "attack" in place, but I preferred to use "cyberattacker" for "attacker" and "adversary."
* All of our "defense" terminology is related to cybersecurity threats, but I did change "defense" to "defense-in-depth cybersecurity" in a couple of spots.
* "guard" :point_right: "safeguard"
* "hang" :point_right: "stop responding"
* "attack surface area" :point_right: "vulnerabilities"
* "locked down" :point_right: "secured"
* BTW ... hitting "i.e."/"e.g." updates along the way ... not bias-free stuff, but we're not supposed to use these. Note that I omitted commas for short parenthetical phrases "(for example to {DO SOMETHING})." A comma in those cases didn't seem significant.
* One "master-detail" item changed to "primary-subordinate."

## Not addressed

There are a few articles that are probably OK on he/his/him/her pronoun things because they refer to specific people with a usually obvious gender. Personally, I'd go with something like "user"/"another user" or "user"/"second user," but these are probably OK. Best to check tho in case anyone wants to update the John/Jane examples:

* https://learn.microsoft.com/en-us/aspnet/core/data/ef-mvc/concurrency?view=aspnetcore-8.0
* https://learn.microsoft.com/en-us/aspnet/core/data/ef-rp/concurrency?view=aspnetcore-8.0&tabs=visual-studio

### CODE REVIEW

File: aspnetcore/blazor/file-downloads.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ When downloading files from a different origin than the app, Cross-Origin Resour
 
 ## Security considerations
 
-Use caution when providing users with the ability to download files from a server. Attackers may execute [Denial of Service (DoS)](/windows-hardware/drivers/ifs/denial-of-service) attacks, [API exploitation attacks](https://developer.mozilla.org/docs/Web/HTML/Element/a#security_and_privacy), or attempt to compromise networks and servers in other ways.
+Use caution when providing users with the ability to download files from a server. Cyberattackers may execute [Denial of Service (DoS)](/windows-hardware/drivers/ifs/denial-of-service) attacks, [API exploitation attacks](https://developer.mozilla.org/docs/Web/HTML/Element/a#security_and_privacy), or attempt to compromise networks and servers in other ways.
```
Comment (wadepickett) [inline aspnetcore/blazor/file-downloads.md:44]:
```suggestion
Use caution when providing users with the ability to download files from a server. Cyber attackers may execute [Denial of Service (DoS)](/windows-hardware/drivers/ifs/denial-of-service) attacks, [API exploitation attacks](https://developer.mozilla.org/docs/Web/HTML/Element/a#security_and_privacy), or attempt to compromise networks and servers in other ways.
```

Comment (guardrex) [reply aspnetcore/blazor/file-downloads.md:44]: The guidelines call for one word, no hyphen.

Comment (wadepickett) [reply aspnetcore/blazor/file-downloads.md:44]: Very minor and feel free to refuse or accept: I think both "cyberattackers"and "cyber attackers" are used in the development community, but I think "cyber attackers" might be used more often.

Comment (guardrex) [reply aspnetcore/blazor/file-downloads.md:44]: I don't think we have a choice given how explicit they are in the style manual. We've broken with them a bit here and there, but I think AP and Chicago are in agreement. My Chicago is buried at the moment tho ... in a box somewhere around here from the last move 😆.

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/test/troubleshoot-azure-iis/includes/troubleshoot-azure-iis2.md [hunk 2]
```diff
@@ -401,7 +401,7 @@ Setting the environment variable for `ASPNETCORE_ENVIRONMENT` is only recommende
 
 If an app is capable of responding to requests, obtain request, connection, and additional data from the app using terminal inline middleware. For more information and sample code, see <xref:test/troubleshoot#obtain-data-from-an-app>.
 
-### Slow or hanging app (IIS)
+### Slow or nonresponding app (IIS)
```
Comment (wadepickett) [inline aspnetcore/test/troubleshoot-azure-iis/includes/troubleshoot-azure-iis2.md:404]:
```suggestion
### Slow or non-responding app (IIS)
```

Comment (guardrex) [reply aspnetcore/test/troubleshoot-azure-iis/includes/troubleshoot-azure-iis2.md:404]: Not per Chicago.

Comment (wadepickett) [reply aspnetcore/test/troubleshoot-azure-iis/includes/troubleshoot-azure-iis2.md:404]: Very minor, take or leave: I think you can go either way technically, but I think non-responding might be a bit more common.

**Review by wadepickett — APPROVED** [review]
Looks great!

File: aspnetcore/test/troubleshoot-azure-iis/includes/troubleshoot-azure-iis2.md (hunk 2 above)
Comment (guardrex) [reply aspnetcore/test/troubleshoot-azure-iis/includes/troubleshoot-azure-iis2.md:404]: "non-" is listed ...

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/non

... and we defer to Chicago per MS guidelines ...

https://learn.microsoft.com/en-us/style-guide/grammar/grammar-and-parts-of-speech

**Pull request merged**